frontend/packages/console-shared/src/selectors/storage.ts

```diff
@@ -0,0 +1,4 @@
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+export const getPvcStorageSize = (pvc: K8sResourceKind): string =>
```

Comment:
Console naming conventions would have `PVC` as uppercase. I would make it clear in the name that it's the request.

```suggestion
export const getRequestedPVCSize = (pvc: K8sResourceKind): string =>
```